Source/EasyNetQ/Extentions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace EasyNetQ
+{
+    public static class Extentions
+    {
+        public static void SafeDispose(this IDisposable disposable)
```

Comment:
Model's dispose method finally call Close with abort=false. So, exception could be thrown.

public void Close(ShutdownEventArgs reason, bool abort)
    {
      ShutdownContinuation shutdownContinuation = new ShutdownContinuation();
      this.ModelShutdown += new EventHandler<ShutdownEventArgs>(shutdownContinuation.OnConnectionShutdown);
      try
      {
        this.ConsumerDispatcher.Quiesce();
        if (this.SetCloseReason(reason))
          this._Private_ChannelClose(reason.ReplyCode, reason.ReplyText, (ushort) 0, (ushort) 0);
        shutdownContinuation.Wait();
        this.ConsumerDispatcher.Shutdown((IModel) this);
      }
      catch (AlreadyClosedException ex)
      {
        if (!abort)
          throw ex;
      }
      catch (IOException ex)
      {
        if (!abort)
          throw ex;
      }
    }